.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,48 @@
+name: Bug Report
+description: Create a bug report for Aliucord itself, to report bugs in plugins please go to our support server or the plugin's repository and make an issue there instead.
```

Comment:
I'd say split the complex sentence into two separate ones since it is a bit long winded.